docs/get-started.md

```diff
@@ -125,20 +128,63 @@ matches:
 
 We're almost there! After every configuration change, **espanso must be restarted**. Open a terminal and type:
 
-```
+```bash
 espanso restart
 ```
 
-Now try to type `:br` anywhere. If you did everything correctly, you should see `Best Regards` appear! 
+Now try to type `:br` anywhere. If you did everything correctly, you should see `Best Regards` appear!
+
+> In version 0.5.1, espanso introduced the `edit` subcommand which makes editing configuration files much easier. Take a look at [Quick Editing](/docs/configuration/#quick-editing) if you are interested.
+
+### Customizing the Toggle Key
```

Comment:
Could we move this section below the `Useful shortcuts` section at the end?

Reply:
There we go! I didn't even notice that at the bottom there, whoops 😅